### DISCUSSION TITLE
Backups not found in UI

### DISCUSSION THREAD

**Discussion by SteveDevDev** [mealie-recipes/mealie#1366] [Q&A] [answered]
Hello,

I'm on version 0.5.6.  I looked at the doc but can't figure out what page the Backup feature is on.

any help?

**Comment by michael-genson** [comment, marked as answer]
It's under the admin dashboard on the right:
![image](https://user-images.githubusercontent.com/71845777/172956275-65956d42-edec-4670-b159-bf458c8219f7.png)

You should be able to go there directly by navigating to `http://my-mealie-domain/admin/dashboard`

**Reply by SteveDevDev** [reply]
I must be blind.  Thank you!
